src/app/components/index/index.component.ts

```diff
@@ -8,14 +8,6 @@ import { Component } from "@angular/core";
 export class IndexComponent {
   constructor() {}
 
-  elements: any = [
-    { id: 1, first: "Mark", last: "Otto", handle: "@mdo" },
-    { id: 2, first: "Jacob", last: "Thornton", handle: "@fat" },
-    { id: 3, first: "Larry", last: "the Bird", handle: "@twitter" }
-  ];
-
-  headElements = ["ID", "First", "Last", "Handle"];
-
   images = [1, 2, 3].map(() => "assets/img/back.jpg");
```

Comment:
better to put it in mock service and take from there

Reply:
Service added